modules/openxr/SCsub

```diff
@@ -45,6 +46,11 @@ elif env["platform"] == "linuxbsd":
 elif env["platform"] == "windows":
     env_thirdparty.AppendUnique(CPPDEFINES=["XR_OS_WINDOWS", "NOMINMAX", "XR_USE_PLATFORM_WIN32"])
 
+env_khrloader = env_thirdparty.Clone()
+if "-fno-exceptions" in env_khrloader["CXXFLAGS"]:
```

Comment:
It's not entirely clear to me why `env_thirdparty` and `env_khrloader` are split here. The idea in the original SCsub is that `env_thirdparty` contains all thirdparty sources while `env_openxr` contains our modules sources. The KHR loader is part of the third party sources.

Reply:
Should be done in the latest version of the PR.